### DISCUSSION TITLE
Were can I read about the new features from the last few updates?

### DISCUSSION THREAD

**Discussion by RikardSwed** [RyotaUshio/obsidian-pdf-plus#167] [Q&A] [answered]
Were can I read about the new features that has come from the different update versions? The reason I ask is because I know the features that excisted a month ago when a tutorial was made on it on youtube. However updates has been done and it would be nice to just read a list of those to understand what the new features are.  Not digging through the whole documentation

**Comment by RyotaUshio** [comment, marked as answer]
The changelog is available from the [releases page](https://github.com/RyotaUshio/obsidian-pdf-plus/releases) of this repository.
Go to the top page of this repository, and then click "Releases" on the right side of the page.
There you can read all the release notes.

**Reply by RikardSwed** [reply]
thanks a lot!
